whois-api/src/test/java/net/ripe/db/whois/api/AbstractIntegrationTest.java

```diff
@@ -96,4 +104,23 @@ protected void removeLog4jAppender() {
     protected String getRequestLog() {
         return stringWriter.toString();
     }
+
+    protected Map<String, String> getRelationCallsFromLinks(final List<Link> links){
+        return links.stream()
+                .filter(link -> {
+                    if (link.getRel() == null){
+                        return false;
+                    }
+                    try {
+                        RelationType.valueOf(link.getRel().toUpperCase());
+                        return true;
+                    } catch (IllegalArgumentException ex) {
+                        return link.getRel().equals("up-active") || link.getRel().equals("top-active");
```

Comment:
That may not be necessary? Also if you are expecting a relationType then up-active and top-active should also be part of enum ?